engines_json/fr.bnf-bibliotheque_nationale_de_france.json

```diff
@@ -0,0 +1,14 @@
+{
+		"_name": "BnF",
```

Comment:
All this data is not needed. Please see e.g https://github.com/egh/zotero-lookup-engines/blob/master/engines_json/epfl.json

All the rest of the data should be generated by the https://github.com/egh/zotero-lookup-engines/blob/master/_item_json.liquid template.